src/Minifiers/BladeMinifier.php

```diff
@@ -27,15 +27,24 @@ class BladeMinifier implements MinifierInterface
     protected $force;
 
     /**
+     * Paths, which should not be minified.
+     *
+     * @var bool
+     */
+    public $ignoredPaths;
```

Comment:
Please don't have a public property here. Pass this information to the compiler directly instead.